src/store/details-panel/details-panel-action.ts

```diff
@@ -13,6 +14,12 @@ export type DetailsPanelAction = UnionOf<typeof detailsPanelActions>;
 
 export const loadDetailsPanel = (uuid: string) => detailsPanelActions.LOAD_DETAILS_PANEL(uuid);
 
-
-
-
+export const toggleDetailsPanel = () => (dispatch: Dispatch) => {
+    // because of material-ui issue resizing details panel breaks tabs.
+    // triggering window resize event fixes that.
+    const detailsPanelAnimationDuration = 500;
```

Comment:
There is also a `SLIDE_TIMEOUT` constant. IMO there should be a one constant that holds animation duration value.